project-communications/academic-updates/annual-health-programme-update.md

```diff
@@ -0,0 +1,184 @@
+Slides on Zenodo. 
+* 09-06-2020: “”Research to empower: co-creating a citizen science platform”, delivered by Georgia Aitkenhead as part of a panel discussion hosted by Kirstie Whitaker for the “Tools, Practices, and Systems” of CogX 2020. 
+Slides on Zenodo. 
+Recording on Youtube.
+* 06-07-2020: “Participatory science to improve autistic people’s lives”, delivered by Kirstie Whitaker. 
+Talk given internally to Fujitsu on working together to build a participatory, citizen science platform.
+Slides on Zenodo.
+* 08-07-2020: “Research to empower: co-creating a citizen science platform", delivered for the Autistica Research Festival 2020 by Kirstie Whitaker, Georgia Aitkenhead, members of Fujitsu and citizen scientists. 
+Slides on Zenodo. 
+Presentation recording on Youtube. 
+* 09-09-2020: The project was presented to the Open Humans community during a community call by Georgia Aitkenhead and a pro bono volunteer. 
+Call recording on YouTube.
+* 02-11-2020: “Participatory citizen science to improve autistic people’s lives”. 
+Presentation given by Kirstie Whitaker for the February Urban Analytics Meeting at The Alan Turing Institute. 
+Slides on Zenodo.
+* 18-11-2020: “Research to Empower: co-creating a citizen science platform” was presented by Georgia Aitkenhead and a citizen scientist for an Autistica webinar on: “How can we move towards a more autism friendly world”. 
+Recording available on Youtube. 
+* 17-12-2020: “Autspaces: creating a participatory, open citizen science platform.” Presentation given by Georgia Aitkenhead and a citizen scientist for the graduation of OLS-2, 2020. 
+Presentation on YouTube.
+* 31-03-2021: “Participatory Citizen Science”, a talk delivered by Georgia Aitkenhead and three citizen scientists for the third cohort of OLS, using AutSPACEs as a case study for participatory citizen science. 
+Slides on Zenodo. 
+Recording on Youtube.
+    
+## Articles/blogs/press releases published (include link/s)
+
+* Project newsletters have been released. 
+Each one has included interviews with project collaborators. 
+They are available on GitHub in the AutisticaCitizenScience repository. 
+The newsletter has 73 subscribers.   
+ 
+* Software/code/tools/methods developed/released
+Methods, protocols, code, and other work as detailed above has been publicly released and is available under an open license on the AutisticaCitizenScience repository and the AutSPACEs repository. 
+ 
+## External engagement, influence, impact (Academic/ industry/ government/ public/ international)
+
+* Our participatory process involves engaging with, supporting, remunerating and in some cases upskilling autistic participants, for instance by teaching GitHub skills, giving a platform for views, and encouraging/supporting public speaking and writing. 
+This is core to our methodology and values.
+* A Twitter account was set up for the project: @AutSpaces. 
```

Comment:
I'm not sure if the @ links to the twitter account. If not I suggest "[Autspaces](https://twitter.com/AutSpaces)